samples/OpenIdConnectSample/Startup.cs

```diff
@@ -158,7 +164,7 @@ await context.Authentication.SignOutAsync(OpenIdConnectDefaults.AuthenticationSc
                 {
                     // This is what [Authorize] calls
                     // The cookie middleware will intercept this 401 and redirect to /login
-                    await context.Authentication.ChallengeAsync();
+                    await context.ChallengeAsync(CookieAuthenticationDefaults.AuthenticationScheme);
```

Comment:
`ChallengeAsync()`
This comment is wrong, this sample has no `/login`. It may have been copied from SocialAuth. This sample should challenge OIDC implicitly.